tests/api_test.py

```diff
@@ -443,9 +443,9 @@ def test_wrong_template(self):
         with pytest.raises(InvalidTypeError):
             templ = ScTemplate()
             templ.triple_with_relation(
-                (ScAddr(0), "_main_node"),
-                (sc_types.EDGE_D_COMMON_CONST, "_const_edge"),
-                (sc_types.LINK_VAR, "_link"),
+                ScAddr(0) >> "_main_node",
```

Comment:
It can be used with curled brackets?

Reply:
Like `(ScAddr(0) >> "_main_node")`? Yes, but because of python syntax

Reply:
okay